apps/src/dance/lottie/LottieDancerUtils.ts

```diff
@@ -334,108 +381,114 @@ export function getAssetById(
   id: string
 ): LottieAssetPrecomp | null {
   return (animationData.assets || []).find(
-    a => a && a.id === id
+    animationAsset => animationAsset && animationAsset.id === id
   ) as LottieAssetPrecomp | null;
 }
 
-/**
- * Hides vector head layers inside the head comp. Also captures a copy of the
- * first matching layer’s transform (ks), so we can reuse it for positioning.
```

Comment:
It is handy having a definition of "Ks" somewhere; I found myself wondering while reading nearby code...

Reply:
Good call. Will re-add!